### PR TITLE
Fix uploading of CI artifacts

### PR DESCRIPTION
**Pull request by heplesser**

### CODE REVIEW

File: .github/workflows/nestbuildmatrix.yml [hunk 1]
```diff
@@ -848,6 +847,5 @@ jobs:
         with:
           name: "${{ matrix.NEST_BUILD_TYPE }}-build-logs-${{ matrix.os }}-${{ matrix.cpp_compiler }}"
           path: |
-            install_manifest.txt
-            **.log
-            build/reports/**
+            build/install_manifest.txt
```
Comment (lekshmideepu) [inline .github/workflows/nestbuildmatrix.yml:850]: Perfect. This then fixes the issue with the warning that those files were not found in the  provided path:

**Review by lekshmideepu — COMMENTED** [review]
👍

File: .github/workflows/nestbuildmatrix.yml (hunk 1 above)
Comment (heplesser) [reply .github/workflows/nestbuildmatrix.yml:850]: @lekshmideepu Github now sees your positive comment as an unresolved, merge-blocking comment. Could you resolve it and approve, unless there are other issues?

**Review by lekshmideepu — APPROVED** [review]
👍

**Review by terhorstd — APPROVED** [review]
Thanks for the fix!
LGTM :+1:

**Pull request merged**